.evergreen/config.yml

```
@@ -346,7 +346,13 @@ functions:
       params:
         shell: bash
         working_dir: ${working_dir|libmongocrypt}
-        script: bash ${workdir}/libmongocrypt/.evergreen/earthly.sh ${args}
+        script: |
+          # Authenticate to artifactory.
+          echo "${artifactory_password}" | docker login --password-stdin --username "${artifactory_username}" artifactory.corp.mongodb.com
+          # TODO(MONGOCRYPT-747): remove `--persist-build=false`. 
+          # Pass `--persist-build=false` to avoid using Docker Hub.
+          # Earthly hardcodes use of docker/dockerfile-copy:v0.1.9 for the CACHE command.
+          bash ${workdir}/libmongocrypt/.evergreen/earthly.sh ${args} --persist-build=false
```

```suggestion
          bash ${workdir}/libmongocrypt/.evergreen/earthly.sh --persist-build=false ${args}
```
Recommend moving flag before user-defined args to avoid undesirable expansion behavior.

---

Unfortunately cannot do. Arguments to the `[build args ...]` follow the `<target ref>`. Placing `--persist-build=false` before results in the error:
```
Error: flag provided but not defined: -persist_build
```